src/sentry/static/sentry/app/views/settings/projectGeneralSettings.jsx

```diff
@@ -473,19 +475,33 @@ class ProjectGeneralSettings extends AsyncView {
             />
           )}
 
-          <JsonForm
-            {...jsonFormProps}
-            title={t('Data Privacy')}
-            fields={[
-              fields.dataScrubber,
-              fields.dataScrubberDefaults,
-              fields.scrubIPAddresses,
-              fields.sensitiveFields,
-              fields.safeFields,
-              fields.storeCrashReports,
-              fields.relayPiiConfig,
-            ]}
-          />
+          {organization.features.includes('datascrubbers-v2') ? (
```

Comment:
We typically use the Feature component for this, as it also handles displaying alternate 'error' states when the feature isn't available.

Reply:
@markstory  I replaced this condition with the `Feature` component. I tested it and it works as expected, but I'm not sure if I'm using the component correctly. It would be nice to improve the`<Feature>`, `<FeatureDisabled>` components documentation in our design system (storybook)